### PR TITLE
Full Node UI pages. Initial development

### PR DESCRIPTION
**Pull request by yhaspel**

### CODE REVIEW

File: app/components/fullNode/FullNodeLoadingPage.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    text: 'Show computer effort'
+  }
+];
+
+const loadingInformationText = "Full node setup may take up to 48 hours. you can leave it running and continue using your device as usual, just don't turn it off.";
+
+class FullNodeLoadingPage extends Component<SeupPageProps, SetupPageState> {
+  render() {
+    return [
+      <div>
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeLoadingPage.js]: remove unnecessary div

File: app/components/fullNode/FullNodeLoadingPage.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+        <FullNodeHeaderText>Full Node Setup</FullNodeHeaderText>
+      </div>,
+      <PageWrapper>
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeLoadingPage.js]: give PageWrapper proper name as it doesn't describe component's usage

File: app/components/fullNode/FullNodeJointStyles.js [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+  }
+  &:active {
+    opacity: 0.6;
+  }
+`;
+
+export const PageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeJointStyles.js]: this is meaningless prop as per component's usage

File: app/components/fullNode/FullNodeLoadingPage.js [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    id: 'showComputerEffort',
+    text: 'Show computer effort'
+  }
+];
+
+const loadingInformationText = "Full node setup may take up to 48 hours. you can leave it running and continue using your device as usual, just don't turn it off.";
+
+class FullNodeLoadingPage extends Component<SeupPageProps, SetupPageState> {
+  render() {
+    return [
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeLoadingPage.js]: this structure
 < FullNodeHeaderText>...</FullNodeHeaderText>
<PageWrapper>
        <LeftPane>{this.renderLeftPane()}</LeftPane>
        <RightPane>{this.renderRightPane()}</RightPane>
</PageWrapper>
repeats itself in 3 components. please DRY

File: app/components/fullNode/FullNodeJointStyles.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+// $FlowStyledIssue
+export const BaseImage = styled.img`
+  width: ${({ width }) => width || 42}px;
+  height: ${({ height }) => height || 42}px;
+`;
+
+export const ImageWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeJointStyles.js]: this prop is unnecessary as per component usage
add align-items: center; and width: 100%; to make inner image to be centered automaticly

File: app/components/fullNode/FullNodeLoadingPage.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+      <PageWrapper>
+        <LeftPane>{this.renderLeftPane()}</LeftPane>
+        <RightPane>{this.renderRightPane()}</RightPane>
+      </PageWrapper>
+    ];
+  }
+
+  renderRightPane = () => (
+    <RightPaneInner>
+      <ImageWrapper>
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeLoadingPage.js]: turn this to basic component Image that accepts only max-height param so the image can be proportional
<ImageWrapper>
        <BaseImage src={thinking} width={132} height={152} />
      </ImageWrapper>

File: app/components/fullNode/FullNodeJointStyles.js [hunk 7]
```diff
@@ -0,0 +1,204 @@
...
+`;
+
+export const LeftPaneInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 100%;
+`;
+
+export const RightHeaderWrapper = styled.div`
```
Comment (IlyaVi) [inline app/components/fullNode/FullNodeJointStyles.js]: any particular reason for having wrapper and then nesting inside RightHeaderText?

File: app/basicComponents/CheckDone.js [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  display: block;
+  height: 8px;
+  width: 10px;
+`;
+
+type Props = {
+  mode?: 'white' | 'green'
+};
+
+const Loader = ({ mode }: { mode?: 'white' | 'green' }) => {
```
Comment (IlyaVi) [inline app/basicComponents/CheckDone.js]: rename to match component's name

File: app/redux/fullNode/actions.js [hunk 9]
```diff
@@ -0,0 +1,14 @@
+// @flow
+export const SET_CAPACITY: string = 'SET_CAPACITY';
+export const RESET_NODE_SETTINGS: string = 'RESET_NODE_SETTINGS';
+
+export const setFullNodeStorage = ({ capacity, drive }: { capacity: ?number, drive: ?string }) => (dispatch: Function) => {
```
Comment (IlyaVi) [inline app/redux/fullNode/actions.js]: why you use dispatch here?

File: app/redux/fullNode/actions.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+  dispatch({ type: SET_CAPACITY, payload: { capacity, drive } });
+};
+
+export function resetNodeSettings() {
+  return {
```
Comment (IlyaVi) [inline app/redux/fullNode/actions.js]: consistency: use ({}) as you used in other files

File: app/routes.js [hunk 11]
```diff
@@ -1,16 +1,47 @@
...
+import { Auth, Wallet, FullNode } from './screens';
 import Root from './screens/Root';
+import StoryBook from './components/StoryBook';
 
 const routes = {
-  HOME: {
-    path: '/auth',
-    component: Auth
+  app: {
+    HOME: {
```
Comment (IlyaVi) [inline app/routes.js]: use array for routes instead of turning them to array of keys and parsing them

File: app/routes.js [hunk 12]
```diff
@@ -1,16 +1,47 @@
...
+    WALLET: {
+      path: '/root/wallet',
+      component: Wallet
+    },
+    FULLNODE: {
+      path: '/root/full-node',
+      component: FullNode
+    }
   }
+  // fullNode: {
```
Comment (IlyaVi) [inline app/routes.js]: remove comment

File: app/screens/fullNode/FullNode.js [hunk 13]
```diff
@@ -0,0 +1,68 @@
...
+type FullNodeState = {};
+
+const Container = styled.div`
+  padding: 58px 90px;
+  width: 100%;
+  height: inherit;
+`;
+
+const routes = {
+  fullNode: {
```
Comment (IlyaVi) [inline app/screens/fullNode/FullNode.js]: same here - use array

File: app/components/fullNode/FullNodeLoadingPage.js (hunk 4 above)
Comment (yhaspel) [reply app/components/fullNode/FullNodeLoadingPage.js]: OK

File: app/components/fullNode/FullNodeJointStyles.js (hunk 7 above)
Comment (yhaspel) [reply app/components/fullNode/FullNodeJointStyles.js]: nope.

File: app/redux/fullNode/actions.js (hunk 9 above)
Comment (yhaspel) [reply app/redux/fullNode/actions.js]: removed.

File: app/screens/fullNode/FullNode.js (hunk 13 above)
Comment (yhaspel) [reply app/screens/fullNode/FullNode.js]: np

File: app/components/localNode/LocalNodeSetupPage.js [hunk 14]
```diff
@@ -0,0 +1,294 @@
...
+
+  render() {
+    return <LocalNodeBase header="Local Node Setup" leftPane={this.renderLeftPane} rightPane={this.renderRightPane} />;
+  }
+
+  renderRightPane = () => (
+    <RightPaneInner>
+      <RightHeaderText>Local Node Setup Instructions</RightHeaderText>
+      {rightPaneSetupModeList.map((setupListItem) => (
+        <SettingModeListItem key={setupListItem.id}>
```
Comment (IlyaVi) [inline app/components/localNode/LocalNodeSetupPage.js]: extract to component

Comment (yhaspel) [reply app/components/localNode/LocalNodeSetupPage.js]: OK

**Pull request merged**